### PR TITLE
Fix arena join permission

### PR DESCRIPTION
**Pull request by VerschuerenTom**
If statement for checking permissions was totally wrong. I fixed it.

### CODE REVIEW

File: src/main/java/plugily/projects/murdermystery/arena/ArenaManager.java [hunk 1]
```diff
@@ -121,9 +121,9 @@ public static void joinAttempt(Player player, Arena arena) {
       }
     }
 
-    if (!plugin.getConfigPreferences().getOption(ConfigPreferences.Option.BUNGEE_ENABLED)
-      && !player.hasPermission(PermissionsManager.getJoinPerm().replace("<arena>", "*"))
-      || !player.hasPermission(PermissionsManager.getJoinPerm().replace("<arena>", arena.getId()))) {
+    if (plugin.getConfigPreferences().getOption(ConfigPreferences.Option.BUNGEE_ENABLED)
```
Comment (VerschuerenTom) [inline src/main/java/plugily/projects/murdermystery/arena/ArenaManager.java:124]: What is this line for checking BUNGEE_ENABLED even doing here? @2Wild4You @montlikadani

**Review by Tigerpanzer02 — COMMENTED** [review]
The bungee option check is needed as it let players join without permission as bungee bases servers only have one minigame and should auto join

**Comment by VerschuerenTom** [comment]
@Tigerpanzer02 Okey. Now it should be correct then. It does the following: 

if NOT (Bungee is enabled OR has permission join.* OR has permission join.<arena> )  THEN return.

**Review by Tigerpanzer02 — APPROVED** [review]
Seems fine now, should check if other plugins got same problem

**Comment by VerschuerenTom** [comment]
@Tigerpanzer02 Others seem to be fine and have an extra ( ) added which makes it correct :).

PR merged and closed.

**Pull request merged**